website/src/html.js

```diff
@@ -1,22 +1,10 @@
 import React from 'react';
-import Gitter from 'react-sidecar';
+// gatsby throws some error when importing just react-sidecar.
+// Maybe because jsx file ext?
+import Gitter from 'react-sidecar/dist-modules/index.js';
```

Comment:
I'm not a big fan of importing minor modules from NPM that we can just do ourselves (and possible output cleaner). If you're good with it, I'd rather just add the Gitter scripts manually. I haven't tested the code below, but it includes everything necessary.

```js
  componentDidMount () {
    ((window.gitter = {}).chat = {}).options = {
      activationElement: '.gitter-open-chat-button',
      room: 'netlify/NetlifyCMS',
    };
  }
```
```js
           <div className="gitter-open-chat-button">Open chat</div>
           <script async defer src="https://sidecar.gitter.im/dist/sidecar.v1.js" />
```

Reply:
We could actually leave off both the "Open chat" `div`, and the `activationElement` line if we wanted to -- those are generated automatically as well.

Reply:
Sounds good! I'll make this change.

Reply:
So you have to do something like this because SSR

```
    if (typeof window !== 'undefined') {
      const Gitter = require('gitter-sidecar');
      new Gitter({
        room: 'netlify/NetlifyCMS',
      });
    }
  }
```

Then change the HTML file to use commonjs modules. Is this really cleaner?

Reply:
Hmm. It's also possible to do something like this in `render`:

```js
    <script
      dangerouslySetInnerHTML={{
        __html: `
         ((window.gitter = {}).chat = {}).options = {
              room: 'netlify/NetlifyCMS'
          }`
      }}
    />
    <script async defer src="https://sidecar.gitter.im/dist/sidecar.v1.js" />
```

I'm fine with whatever you want, just go for it. 😃

Reply:
I'd rather just leave it honestly